util/update/run.go

```diff
@@ -49,5 +52,29 @@ func Run(path string) error {
 	// update the model
 	update := newUpdate()
 
+	newVersion := getCurrentVersion(path)
+
+	fmt.Printf("Updated to version: %s", newVersion)
```

Comment:
```go
fmt.Printf("\nUpdated to version: %s\n\n", newVersion)
fmt.Println("Check out the release notes here:")
fmt.Println("https://github.com/nanobox-io/nanobox/blob/master/CHANGELOG.md")
```

Reply:
With this, you could also eliminate the need for the `newVersion` variable by just calling `getCurrentVersion`, since it wouldn't be used for printing the release url.